x-pack/plugins/security_solution/public/flyout/right/components/entities_overview.tsx

```diff
@@ -52,7 +52,15 @@ export const EntitiesOverview: React.FC = () => {
               defaultMessage="Entities"
             />
           ),
-          callback: goToEntitiesTab,
+          link: {
+            callback: goToEntitiesTab,
+            tooltip: (
+              <FormattedMessage
+                id="xpack.securitySolution.flyout.right.insights.entities.entitiesTooltip"
+                defaultMessage="View all entities"
```

Comment:
```suggestion
                defaultMessage="Show all entities"
```

Reply:
good catch, I don't know how I missed that one sorry!

Reply:
All good! It was a minor change :D